snapcraft/storeapi/__init__.py

```diff
@@ -387,6 +399,63 @@ def snap_release(self, snap_name, revision, channels):
 
         return response_json
 
+    def generate_snap_build(self, account_info, snap_name,
```

Comment:
See the need of the account info inside generate_snap_build(), though I will see how I can refactor it.